assets/style.css

```diff
@@ -0,0 +1,1604 @@
+  flex-wrap: wrap;
+  width: 100%;
+  height: 100%;
+}
+
+.my-col img {
+  height: 70%;
+  padding-top: 10px;
+  width: 100%;
+  cursor: pointer;
+}
+
+.my-col h3:hover {
+  -webkit-text-decoration: underline #f72210;
+  text-decoration: underline #f72210;
+}
+
+.editors {
+  max-width: 1300px;
+  border-bottom: 1px solid lightgray;
+}
+
+.ed-col {
+  cursor: pointer;
+  margin-bottom: 30px;
+}
+
+.ed-col h2:hover {
+  -webkit-text-decoration: underline #f72210;
+  text-decoration: underline #f72210;
+}
+
+.ed-col p {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  cursor: text;
+}
+
+.last-articles {
+  max-width: 1300px;
+  margin-top: 30px;
+}
+
+.ed-col1 {
+  margin-bottom: 10px;
+  border-bottom: 1px solid lightgray;
+  padding-bottom: 30px;
+}
+
+.ed-col1 img {
+  cursor: pointer;
+}
+
+.ed-col1 p {
+  font-size: 22px;
+  font-weight: bold;
+}
+
+.footer-links p {
+  font-size: 14px;
+  color: #666;
+  padding-right: 10px;
+}
+
+.footer-editions p {
+  font-size: 14px;
+  color: #666;
+  padding: 0 10px;
+  border-right: 1px solid lightgray;
+}
+
+.ed-col1 p:hover {
+  cursor: pointer;
+  -webkit-text-decoration: underline #f72210;
+  text-decoration: underline #f72210;
+}
+
+.tesla {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  border-bottom: 1px solid lightgrey;
+  padding-top: 10px;
+}
+
+.tesla img {
+  margin-bottom: 10px;
+}
+
+.tesla h4 {
+  font-size: 17px;
+  font-weight: bold;
+  width: 70%;
+  margin-left: 15px;
+}
+
+.tesla h4:hover {
+  -webkit-text-decoration: #f72210 underline;
+  text-decoration: #f72210 underline;
+  cursor: pointer;
+}
+
+.tsl-last {
+  border-bottom: none;
+}
+
+.sub-news-title {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin: 10px 20px;
+}
+
+.sub-news-cont {
+  width: 100%;
+  max-width: 1330px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  border-bottom: 1px solid #e0dede;
+  margin-bottom: 30px;
+}
+
+.suscribe {
+  background: #f4f5f6;
+  height: 317px;
+  width: 64%;
+  padding: 30px;
+  margin: 0 20px 20px 20px;
+}
+
+.suscribe img {
+  float: left;
+  margin-right: 30px;
+}
+
+.suscribe button {
+  background: #f72210;
+  width: 17%;
+  height: 38px;
+  border-radius: 3px;
+  color: white;
+  border: none;
+  font-weight: bold;
+  margin-top: 20px;
+}
+
+.newsletter {
+  width: 30%;
+  height: 317px;
+  border: 1px solid lightgrey;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.news-text {
+  width: 53%;
+  height: 317px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.news-text button {
+  background: #f72210;
+  width: 80%;
+  height: 38px;
```

Comment:
You are repeating yourself here, you have alredy use this `heigth: 38px;` on line 287, 1087, 1123 :point_right: create a utility class which have this property and apply it wherever you need this property